web/spacewalk-web.spec

```diff
@@ -63,7 +63,7 @@ but it does generate a number of sub-packages.
 
 %package -n susemanager-web-libs
 Summary:        Vendor bundles for spacewalk-web
-License:        0BSD AND BSD-3-Clause AND LGPL-3.0-or-later AND MIT AND MPL-2.0
+License:        0BSD and BSD-3-Clause and LGPL-3.0-or-later and MIT and MPL-2.0
```

Comment:
At first I thought this is weird but then I realized this is due to my recent change to the script that fills the spec file. In the end the case doesn't matter here, so I think it's okay

Reply:
I have a PR that addresses this, for now I needed to rebuild it so the Github Action validation passes. https://github.com/uyuni-project/uyuni/pull/3815